monai/data/dataset.py

```diff
@@ -458,6 +479,7 @@ def _cachecheck(self, item_transformed):
         if self._read_env is None:
             self._read_env = self._fill_cache_start_reader()
         with self._read_env.begin(write=False) as txn:
+            print("!!!!!!!!", item_transformed)
```

Comment:
Print statement shouldn't be here?

Reply:
sorry for the typo, removed it in the latest commit.

Thanks.